src/icm20948_reg.hpp

```diff
@@ -19,136 +19,129 @@
 
 namespace hal::icm {
 
-
-static constexpr hal::byte ICM20948_ADDRESS = 0x69;
-static constexpr hal::byte AK09916_ADDRESS = 0x0C;
+static constexpr hal::byte icm20948_address = 0x69;
+static constexpr hal::byte ak09916_address = 0x0C;
 
 /* Registers ICM20948 USER BANK 0*/
-static constexpr hal::byte ICM20948_WHO_AM_I = 0x00;
-static constexpr hal::byte ICM20948_USER_CTRL = 0x03;
-static constexpr hal::byte ICM20948_LP_CONFIG = 0x05;
-static constexpr hal::byte ICM20948_PWR_MGMT_1 = 0x06;
-static constexpr hal::byte ICM20948_PWR_MGMT_2 = 0x07;
-static constexpr hal::byte ICM20948_INT_PIN_CFG = 0x0F;
-static constexpr hal::byte ICM20948_INT_ENABLE = 0x10;
-static constexpr hal::byte ICM20948_INT_ENABLE_1 = 0x11;
-static constexpr hal::byte ICM20948_INT_ENABLE_2 = 0x12;
-static constexpr hal::byte ICM20948_INT_ENABLE_3 = 0x13;
-static constexpr hal::byte ICM20948_I2C_MST_STATUS = 0x17;
-static constexpr hal::byte ICM20948_INT_STATUS = 0x19;
-static constexpr hal::byte ICM20948_INT_STATUS_1 = 0x1A;
-static constexpr hal::byte ICM20948_INT_STATUS_2 = 0x1B;
-static constexpr hal::byte ICM20948_INT_STATUS_3 = 0x1C;
-static constexpr hal::byte ICM20948_DELAY_TIME_H = 0x28;
-static constexpr hal::byte ICM20948_DELAY_TIME_L = 0x29;
-static constexpr hal::byte ICM20948_ACCEL_OUT =
+static constexpr hal::byte icm20948_who_am_i = 0x00;
```

Comment:
```suggestion 
constexpr hal::byte icm20948_who_am_i = 0x00;
```

I do not believe that `static` is required for these. Other code uses this, but is not desired and should be removed.